composer.json

```diff
@@ -40,7 +40,8 @@
     "browscap/browscap-php": "dev-master",
     "mikey179/vfsStream": "^1.6",
     "squizlabs/php_codesniffer": "^2.8",
-    "friendsofphp/php-cs-fixer": "^1.11"
+    "friendsofphp/php-cs-fixer": "^1.11",
+    "seld/jsonlint": "^1.5"
```

Comment:
I've realised `composer.lock` is gitignored - please can we remove that from gitignore and commit along with the repo. This is a project, not a library so we should have locked down deps to avoid issues.